src/utils/utils.js

```diff
@@ -0,0 +1,54 @@
+export const MAX_GUESSES = 6;
```

Comment:
We could maybe put this back into the word duel main file and instead pass it as a second argument into `isMaxGuesses`